diff --git a/Hands-On Labs/Using PostgreSQL/create_table.py b/Hands-On Labs/Using PostgreSQL/create_table.py
index c6fee69..2ad094a 100644
--- a/Hands-On Labs/Using PostgreSQL/create_table.py	
+++ b/Hands-On Labs/Using PostgreSQL/create_table.py	
@@ -22,7 +22,14 @@ def create_table():
     );   
     """
 
-    pass # student code goes here
+    conn = get_database_connection()
+    curs = conn.cursor()
+    curs.execute(create_table)
+    conn.commit()
+    curs.close()
+    conn.close()
+    pass
+
 
 
 def populate_table():
